File: AimPlugin3.0.4/AIM.Annotation/Template/OrdinalLevel.cs
#region License

//L
// 2007 - 2013 Copyright Northwestern University
//
// Distributed under the OSI-approved BSD 3-Clause License.
// See http://ncip.github.com/annotation-and-image-markup/LICENSE.txt for details.
//L

#endregion

using System.Xml.Serialization;
using System.Collections.Generic;

namespace AIM.Annotation.Template
{
	public class OrdinalLevel
	{
		[XmlAttribute("integerValue")]
		public int IntegerValue { get; set; }

		[XmlAttribute("description")]
		public string Description { get; set; }
	}
}

File: ATB_1.0_src/src/main/java/edu/stanford/isis/atb/domain/template/OrdinalLevel.java
/*
*  2010 – 2012 Copyright Northwestern University and Stanford University 
*
*  Distributed under the OSI-approved BSD 3-Clause License.
*  See http://ncip.github.com/annotation-and-image-markup/LICENSE.txt for details.
*/



package edu.stanford.isis.atb.domain.template;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Root;

/**
 * @author Vitaliy Semeshko
 */

@Root(name="OrdinalLevel", strict=false)
public class OrdinalLevel extends AbstractRemovableElement {

	@Attribute
	private int integerValue;
	
	@Attribute(required=false)
	private String description;

	public int getIntegerValue() {
		return integerValue;
	}

	public void setIntegerValue(int integerValue) {
		this.integerValue = integerValue;
	}

	public String getDescription() {
		return description;
	}

	public void setDescription(String description) {
		this.description = description;
	}
	
	@Override
	public String toString() {
		return "Ordinal Level: " + integerValue;
	}
	
}
